script.js

```diff
@@ -0,0 +1,90 @@
+const dinnerOptions = document.getElementById('dinnerOptions');
+//label the element id dinnerOptions as the constant dinnerOptions
+
+const firstName = form.firstName;
+//label the constant firstName from the constant form
+const lastName = form.lastName;
+//label the constant lastName from the constant form
+
+const age = document.getElementById('age');
+//label the element id age as the constant age
+
+meal.addEventListener('select', validate, false);
+//adding event listener to wait for meal option to be selected
+function checkMeal(event){
+    if(meal == breakfast){
+        mealOptions = breakfastOptions;
+        mealOptions.breakfastOptions.style.display = 'block';
+    }else if(meal == lunch){
+        mealOptions = lunchOptions;
+        mealOptions.lunchOptions.style.display = 'block';
+    }else if(meal == dinner){
+        mealOptions = dinnerOptions;
+        mealOptions.dinnerOptions.style.display = 'block';
+    }
+    }
+    //if then else function created to check the meal picked and display the meal options for the selected meal
+
+firstName.addEventListener('blur', validate, false);
+//adding event listener to wait for the firstName field to lose focus
+function validateInline1(event){
+    const firstLetter1 = form.firstName[0];
+    event.firstLetter1.toUpperCase();       
+}
+//validateInline1 function created to capitalize the first letter of the first name entered in the form
+lastName.addEventListener('blur', validate, false);
+//adding event listener to wait for the lastName field to lose focus
+function validateInline2(event){
+    const firstLetter2 = form.lastName[0];
+    event.firstLetter2.toUpperCase();
+}
+//validateInline2 function created to capitalize the first letter of the last name entered in the form
+
+age.addEventListener('blur', validate, false);
+//adding event listener to wait for the age field to lose focus
+function checkInputNumbers(event){ 
+var letters = /^[0-9a-zA-Z]+$/;
+if(event.form.age.value.match(letters)){
+    alert('Please input only a number for age!');
+    return true;
+}else{
+    return false;
+}
+}
+//function checkInputNumbers created to check if the input for age has numbers only if letters have been entered an alert will pop up asking the user to only input numbers
+
+form.addEventListener('submit', validate, false);
+//adding event listener to wait for the button to be clicked to submit the form
+
+function disableSubmit(event){
+    if(event.target.value === ''){
+        document.getElementById('submitButton').disabled = true;
+    }else{
+        document.getElementById('submitButton').disabled = false;
+    }
+}
+//function disableSubmit added to disable the submit button if the form has not been filled out
+
+//I am struggling to find the correct way to put this together so it functions, I thought I understood the concepts in the chapter but applying them is not giving me a working result after many different attempts. 
```

Comment:
I would love to hear this from you ealier in the week so that we could meet and put the pieces together. Get an early start this week and when you run into problems - e-mail me a link to your repo or let me know when we could get together on Zoom to go over things. You are so close! I think if we talk through a few things as you are building your solutions - this will come far easier for you 😀